pkgs/development/python-modules/geopy/default.nix

```
@@ -1,31 +1,33 @@
 { stdenv
 , buildPythonPackage
-, fetchPypi
-, isPy27
+, fetchFromGitHub
+, isPy3k
 , mock
 , tox
 , pylint
+, geographiclib
 }:
 
 buildPythonPackage rec {
-  pname = "geopy";
-  version = "1.20.0";
-  disabled = !isPy27;
+    pname = "geopy";
+    version = "2.0-unstable-2019-11-10";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "9419bc90ee6231590c4ae7acf1cf126cefbd0736942da7a6a1436946e80830e2";
-  };
+    disabled = !isPy3k; # only Python 3
+    doCheck = false; # too many checks
```

```suggestion
    doCheck = false; # Needs network access
```
I tried to run them because "too many checks" doesn't sound like a good excuse not to run them unless they take a considerable amount of time.

---

Fair enough, this is just what the comment from the original derivation sounded like to me, my bad